tests/Generator.hs

```diff
@@ -97,3 +101,9 @@ mailboxMessages = MailboxMessage <$> sides <*> phases <*> Gen.maybe messageIDs <
 
 welcomeMessages :: MonadGen m => m WelcomeMessage
 welcomeMessages = WelcomeMessage <$> Gen.maybe (Gen.text (Range.linear 0 1024) Gen.unicode) <*> Gen.maybe (Gen.text (Range.linear 0 1024) Gen.unicode)
+
+offerMessages :: MonadGen m => m F.Offer
+offerMessages = Gen.choice
+  [ F.Message <$> (Gen.text (Range.linear 0 1024) Gen.unicode)
+  , F.File <$> toS <$> (Gen.text (Range.linear 0 100) Gen.alphaNum)  <*> (fromIntegral <$> (Gen.int (Range.linear 0 maxBound)))
```

Comment:
I wonder if there are any concrete size limits in practice or if the server implementation will just take messages up to what its memory can handle.  There probably _should_ be some size limits if there aren't...  Maybe an issue to file against the Python implementation?

Reply:
Good point. I will try finding that out from the python code base. Thanks!